javascript/javascript1/week2/homework/event-app.js

```diff
@@ -0,0 +1,11 @@
+// Exercise 2 - Event application.
+
+function getEventWeekday(daysFromToday) {
+  const weekday = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
```

Comment:
For `newDate().getDay()` to be correct, you need to count as the week starts on Sunday.
```suggestion
  const weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
```

The easiest way to validate this would be to call the function with `getEventWeekday(0)` and see if it shows today's day. If not, then the logic is not correct.

You can read about it here too - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDay